setversion.sh

```diff
@@ -88,12 +103,21 @@ else
     VERSION_SUFFIXED=${OPT_VERSION}
 fi
 
+SHORT_VERSION=`echo ${OPT_VERSION} | cut -d. -f1,2`
 
 echo -e "${YELLOW}*** SETTING VERSION NAME TO ${OPT_VERSION}, VERSION CODE TO ${OPT_CODE}, IS DEV = ${OPT_IS_DEV} ***${NC}"
 
 echo "*** Updating package.json ***"
 update_package_json "./package.json" "${OPT_VERSION}"
 
+echo "*** Updating build.gradle files ***"
+update_build_gradle "./libs/SalesforceAnalytics/build.gradle" "${OPT_VERSION}"
```

Comment:
One more file to add -> top-level `build.gradle` at the root of the repo. That doesn't have a `vcsTag` though. It looks like this:
```
allprojects {
  group = 'com.salesforce.mobilesdk'
  version = '7.1.0'
  repositories {
    maven {
      // All of React Native (JS, Android binaries) is installed from npm
      url "$rootProject.projectDir/libs/SalesforceReact/node_modules/react-native/android"
    }
    google()
    jcenter()
  }
}
```